app/src/main/assets/indoorMapControl.js

```diff
@@ -1,72 +1,119 @@
 var map;
 
 var floormapGroup;
-var markerGroup;
+var roomMarkerGroup;
+var pathMarkerGroup;
 var pathGroup;
 
 function initmap() {
-    map = new L.Map('map', {
-        crs: L.CRS.Simple,
-        minZoom: -1,
-        maxZoom: 1,
-        attributionControl: false, //Remove Attribution on bottom right
-        zoomControl: false //Remove "+" and "-" button on top left
-    });
-    floormapGroup = L.layerGroup().addTo(map);
-
-    //Listener Registration
-    map.on('click', function(ev) {
-        console.log(JSON.stringify(ev.latlng));
-    });
+	map = new L.Map('map', {
+		crs: L.CRS.Simple,
+		minZoom: -2,
+		maxZoom: 1,
+		attributionControl: false, //Remove Attribution on bottom right
+		zoomControl: false //Remove "+" and "-" button on top left
+	});
+	//Image Groups
+	floormapGroup = L.layerGroup().addTo(map);
+
+	//Path Groups
+	pathGroup = L.layerGroup().addTo(map);
+
+	//Marker Groups
+	roomMarkerGroup = L.layerGroup().addTo(map);
+	pathMarkerGroup = L.layerGroup().addTo(map);
+
+	//Listener Registration
+	/*map.on('click', function(ev) {
+		console.log(JSON.stringify(ev.latlng));
+	});*/
 };
 
 function loadMapImage(path) {
-    //Indoor Map
-    floormapGroup.clearLayers();
-
-    var bounds = [
-        [0, 0],
-        [1000, 1000]
-    ];
-    var imageOptions = {
-        interactive: true
-    };
-    floormapGroup.addLayer(L.imageOverlay(path, bounds, imageOptions));
-    map.fitBounds(bounds);
+	//Indoor Map
+	floormapGroup.clearLayers();
+
+	var bounds = [
+		[0, 0],
+		[1989, 2196]
+	];
+	var imageOptions = {
+		interactive: true
+	};
+	floormapGroup.addLayer(L.imageOverlay(path, bounds, imageOptions));
+	map.fitBounds(bounds);
+};
+
+function generateWalkablePath(pointArray) {
+	var points = [];
+
+	for (var i = 0; i < pointArray.length; i++) {
+		var pointRaw = pointArray[i];
+
+		var pointLat = 1989 - pointRaw.lat;
```

Comment:
1989, can you please explain what the value is ? or use the "bounds" variable here if that's what you are getting.